cmd/bosun/web/expr.go

```diff
@@ -66,6 +67,22 @@ func Expr(t miniprofiler.Timer, w http.ResponseWriter, r *http.Request) (interfa
 	return ret, nil
 }
 
+// presentError formats the error in a way appropriate for a web user to consume
+func presentError(err error) string {
+	switch err := err.(type) {
+	case *graphite.RequestError:
```

Comment:
I think it's wrong to have this package depend on the graphite package. Need another solution.